dev/NumberBox/NumberBox.cpp

```diff
@@ -45,6 +45,20 @@ NumberBox::NumberBox()
     LostFocus({ this, &NumberBox::OnNumberBoxLostFocus });
 
     SetDefaultStyleKey(this);
+    SetDefaultInputScope();
+}
+
+void NumberBox::SetDefaultInputScope()
+{
+    // Sets the default value of the InputScope property.
+    // Note that InputScope is a class that cannot be set to a default value within the IDL.
+    winrt::InputScopeName inputScopeName = winrt::InputScopeName(winrt::InputScopeNameValue::Number);
```

Comment:
we prefer using auto and const for the local variable types.

Reply:
Not a problem, should I pursue this now though or wait for potential work-arounds setting in the default style?

Reply:
I just did it now so hopefully we can close this. There is no response on work-arounds for the XAML bugs.